modules/identity/src/Volo.Abp.Identity.Blazor/Pages/Identity/UserManagement.razor.cs

```diff
@@ -228,16 +229,21 @@ protected override ValueTask SetToolbarItemsAsync()
         return base.SetToolbarItemsAsync();
     }
 
-    protected virtual void ChangePasswordTextRole(TextRole? textRole)
+    protected virtual void ChangePasswordTextRole(TextRole? textRole, bool? isShowPassword = null)
```

Comment:
no need `isShowPassword` parameter here. Can use `IsShowPassword` property directly in this function

Reply:
It's simpler but not a safe behaviour

Reply:
It seems `isShowPassword` isn't necessary as a method parameter. That value is already known in the class.